src/declaration.c

```diff
@@ -375,7 +375,7 @@ void AliasDeclaration::semantic(Scope *sc)
     if (s && ((s->getType() && type->equals(s->getType())) || s->isEnumMember()))
         goto L2;                        // it's a symbolic alias
 
-    type = type->addStorageClass(storage_class);
+    type = type->addSTC(storage_class);
```

Comment:
In here, type is not yet semantic analysed. Using `Type::addStrageClass` is valid for the already analyzed type. On the other hand, `Type::addSTC` is for the yet not analyzed type (therefore it's normally used in parsing stage, and rarely in template instantiation process).